manifests/t/TechPowerUp/ThrottleStop/9.7/TechPowerUp.ThrottleStop.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.11.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: TechPowerUp.ThrottleStop
+PackageVersion: '9.7'
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ThrottleStop.exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-12-26
+Installers:
+- Architecture: x86
+  InstallerUrl: https://uk2-dl.techpowerup.com/files/A4niVY7vkYprVE0g3cO3AA/1750980933/ThrottleStop_9.7.zip

```

Comment:
```suggestion
  InstallerUrl: https://uk2-dl.techpowerup.com/files/ThrottleStop_9.7.zip
```

Reply:
You were right, https://github.com/microsoft/winget-pkgs/issues/32777#issuecomment-953652832 shows the `winget/xx` user-agent lets winget reach the TechPowerUp servers without a 404. 

Thanks for the suggestion!
 
```
curl --head --user-agent "winget/xx" https://us2-dl.techpowerup.com/files/ThrottleStop_9.7.zip
```